x-pack/plugins/secops/public/components/page/hosts/host_summary/index.tsx

```diff
@@ -71,37 +72,35 @@ export const getEuiDescriptionList = (
   host: HostItem | null,
   startDate: number,
   endDate: number
-): JSX.Element => {
-  return (
-    <EuiDescriptionList type="column" compressed>
-      {Object.entries(fieldTitleMapping).map(([field, title]) => {
-        const summaryValue: string | string[] | null = get(field, host);
-        return (
-          <React.Fragment key={field}>
-            <EuiDescriptionListTitle>{title}</EuiDescriptionListTitle>
-            {/*Using EuiDescriptionListDescription throws off sizing of Draggable*/}
-            <div>
-              {isArray(summaryValue)
-                ? summaryValue.map((value: string) =>
-                    createDraggable(value, field, startDate, endDate, dateFields)
-                  )
-                : createDraggable(summaryValue, field, startDate, endDate, dateFields)}
-            </div>
-          </React.Fragment>
-        );
-      })}
-    </EuiDescriptionList>
-  );
-};
+): JSX.Element => (
+  <EuiDescriptionList type="column" compressed>
+    {Object.entries(fieldTitleMapping).map(([field, title]) => {
+      const summaryValue: string | string[] | null = get(field, host);
+      return (
+        <React.Fragment key={field}>
+          <EuiDescriptionListTitle>{title}</EuiDescriptionListTitle>
+          {/*Using EuiDescriptionListDescription throws off sizing of Draggable*/}
+          <div>
+            {isArray(summaryValue)
+              ? summaryValue.map((value: string) =>
+                  createDraggable(value, field, startDate, endDate, dateFields)
+                )
+              : createDraggable(summaryValue, field, startDate, endDate, dateFields)}
+          </div>
+        </React.Fragment>
+      );
+    })}
+  </EuiDescriptionList>
+);
```

Comment:
note: I only converted this to an implicit `return`. It just shows everything as changed since all the indentation changed on this one.